docs/adr/0006-monitoring-structure.md

```diff
@@ -0,0 +1,55 @@
+# Application Monitoring in Operate First using Prometheus
+
+* Status: accepted
+* Deciders: 4n4nd, anishasthana, hemajv, HumairAK, tumido
+* Date: 2021-01-06
+
+Technical Story: [issue-1](https://github.com/operate-first/blueprint/issues/8)
+                 [issue-2](https://github.com/operate-first/apps/issues/57)
+
+## Context and Problem Statement
+
+Since there will be multiple services deployed in the Operate First environment
+(ex. Jupyterhub, Argo, Superset, etc), we need to be able to monitor them. <br>
```

Comment:
We're not calling out ODH vs non-ODH here

Reply:
We should be able to monitor them all right? ODH services as well non-ODH services like Observatorium

Reply:
All of the services in this list are ODH, so someone _might_ think it's just caring about ODH.

Reply:
I'll add a non-ODH example